### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by ninpnin**

### CODE REVIEW

File: pyriksdagen/metadata.py [hunk 1]
```diff
@@ -347,7 +392,10 @@ def load_Corpus_metadata(metadata_folder=None):
 
 	# Remove redundancy and split file
 	corpus = corpus.drop_duplicates()
+	print(len(corpus))
```
Comment (ninpnin) [inline pyriksdagen/metadata.py:395]: Are these necessary?

File: pyriksdagen/utils.py [hunk 2]
```diff
@@ -335,3 +333,7 @@ def get_data_location(partition):
     d["motions"] = os.environ.get("MOTIONS_PATH", "data")
     d["metadata"] = os.environ.get("METADATA_PATH", "data")
     return d[partition]
+
+
+def where_am_i():
```
Comment (ninpnin) [inline pyriksdagen/utils.py:338]: What is this for?

Comment (BobBorges) [reply pyriksdagen/utils.py:338]: need it to fetch the queries

File: pyriksdagen/metadata.py (hunk 1 above)
Comment (BobBorges) [reply pyriksdagen/metadata.py:395]: no, should have removed them

Comment (BobBorges) [reply pyriksdagen/metadata.py:395]: will remove

File: pyriksdagen/utils.py (hunk 2 above)
Comment (BobBorges) [reply pyriksdagen/utils.py:338]: image

File: pyriksdagen/metadata.py (hunk 1 above)
Comment (BobBorges) [reply pyriksdagen/metadata.py]: [2f568b0](https://github.com/swerik-project/pyriksdagen/pull/1/commits/2f568b0f31086ef8ba88a8b6677070bcfc10c12e)

**Comment by ninpnin** [comment]
there's a better way to do it

File: pyriksdagen/utils.py (hunk 2 above)
Comment (BobBorges) [reply pyriksdagen/utils.py]: [52721c1](https://github.com/swerik-project/pyriksdagen/pull/1/commits/52721c1e18f9c4afafd5cfacadebcad86666f0c2)

**Pull request merged**